### DISCUSSION TITLE
In the latest version, all machines show as down in the object list

### DISCUSSION THREAD

**Discussion by even0306** [ccfos/nightingale#1196] [Q&A] [answered]
所有机器状态是down，cpu等也都是0。机器的时间再三检查，是准的，而且大盘都是数据正常的，只有对象列表里面没有数据

**Comment by UlricQin** [comment, marked as answer]
截图证明

**Reply by even0306** [reply]
![image](https://user-images.githubusercontent.com/58162761/195797688-aba0b77f-fdd6-41bd-8ee7-08f6bfe5826c.png)
![image](https://user-images.githubusercontent.com/58162761/195797954-15f1234d-1465-4812-b000-4a9ca75e3da4.png)

**Comment by UlricQin** [comment]
没有证明你的时间是一致的。浏览器的时间、时序库的时间、目标机器的时间

**Reply by even0306** [reply]
![image](https://user-images.githubusercontent.com/58162761/195811095-a44fe675-f484-45e9-8907-7dd3bfc21d81.png)
![image](https://user-images.githubusercontent.com/58162761/195811175-25a586c4-d57f-4745-a22b-1276fca6ea2f.png)
![image](https://user-images.githubusercontent.com/58162761/195811373-ae5ed497-10b8-4937-b78e-6a5b3c75b773.png)
![image](https://user-images.githubusercontent.com/58162761/195813600-da69ff3c-5a2d-426b-b334-2e9ffd8baa47.png)

**Reply by even0306** [reply]
都和本地时间一致，时间是准的，如果是毫秒级的时差那我这没有看出来。

**Reply by UlricQin** [reply]
<img width="430" alt="image" src="https://user-images.githubusercontent.com/2769466/195817085-4e11d132-8052-4f41-8a64-9147f4829bb6.png">
这个间隔了10分钟，是正常的？

**Reply by even0306** [reply]
这是我两张截图截的时候差了10分钟，有点其他事情。对比右下角电脑本地时间，是准的

**Comment by UlricQin** [comment]
`100-max(max_over_time(mem_available_percent{ident=~"(192.168.1.41)"}[2m])) by (ident)` 去即时查询页面，查询这个promql，使用table视图，截图看看

**Reply by even0306** [reply]
![image](https://user-images.githubusercontent.com/58162761/195991500-572a0307-587e-4082-a1f2-bcb0ffdddd12.png)

**Reply by UlricQin** [reply]
Table视图!

**Reply by even0306** [reply]
![image](https://user-images.githubusercontent.com/58162761/195993374-98b31ae9-4e79-41b8-92fd-87057e773c08.png)

**Reply by UlricQin** [reply]
这是查看的Default集群的数据？

**Comment by UlricQin** [comment]
把1.41换成1.21在Table视图查一下，换成1.22再查一下，如果都能查到，那整体是没有问题的，估计是某个地方有个小问题没有注意到，摸不到你的环境，目前我没有其他思路

**Reply by even0306** [reply]
![image](https://user-images.githubusercontent.com/58162761/196019024-28c43671-caa9-4944-9765-20114800d82f.png)
![image](https://user-images.githubusercontent.com/58162761/196019035-f464d3a0-e0cd-4263-8538-19aee7c4569d.png)
21和22都有数据，所以很奇怪，不知道问题出哪里，因为公司内网服务器，不能给直接连。如果有其他思路可以继续回，我配合截图排查

**Comment by lsy1990** [comment]
你是否是用的k8s？ 集群是不是部署在k8s上？

**Reply by lsy1990** [reply]
你要确保，每个k8s 是独立集群，那个集群名不能一样，因为不同k8s，虚拟化网络用vxlan，是不通的。
总之，你要保障每个集群集群名唯一，且一定有一个leader

**Reply by even0306** [reply]
没有使用k8s，也没有做任何集群和高可用

**Comment by leilifengxing** [comment]
我也遇到同样的问题，作者解决了吗

**Reply by even0306** [reply]
还没有。目前还是down中

**Reply by leilifengxing** [reply]
跟grafana-agent中的noede_exporter时区有关吗

**Reply by even0306** [reply]
有关的，但我全部一致

**Comment by UlricQin** [comment]
用二进制部署吧，别用容器了，就不会有这些奇奇怪怪的问题了

**Reply by leilifengxing** [reply]
就是物理机上部署的，不是容器

**Comment by UlricQin** [comment]
时区要相同，甚至时间也要一致

**Reply by even0306** [reply]
全部一致
